travel-planner/travel-planner-ui/src/Components/Maps/Maps.jsx

```diff
@@ -54,6 +84,37 @@ export default function Maps({
 			}
 		}, [update]);
 
+		useEffect(() => {
+			// This section displays the directions.
+			if (directionMarkers.length == 2) {
+				var directionsService = new google.maps.DirectionsService();
+				// var directionsRenderer = new google.maps.DirectionsRenderer();
+
+				var request = {
+					origin: directionMarkers[0],
+					destination: directionMarkers[1],
+					travelMode: "DRIVING",
+				};
+
+				directionsService.route(request, (result, status) => {
+					// send the request to get the directions
+					console.log("Request: ", request);
```

Comment:
it appear this is only for local debugging, not production debugging. So I recommend removing.